rest/src/main/kotlin/builder/auditlog/AuditLogGetRequestBuilder.kt

```diff
@@ -22,5 +22,10 @@ class AuditLogGetRequestBuilder : RequestBuilder<AuditLogGetRequest> {
      */
     var before: Snowflake? = null
 
-    override fun toRequest(): AuditLogGetRequest = AuditLogGetRequest(userId, action, before, 100)
+    /**
+     * How many entries are returned (default 50, minimum 1, maximum 100).
+     */
+    var limit: Int = 100
```

Comment:
the initial value of the property and the default value in the documentation are different, which could be confusing

Reply:
it defaulted to 100 in Kord before, judging whether this was the right choice isn't part of this PR

Reply:
I would suggest updating the documentation to something like:
>How many entries are returned (default 50, minimum 1, maximum 100). ([discord docs](https://discord.com/developers/docs/resources/audit-log#get-guild-audit-log))
100 by default.

Reply:
I see, well now this is merged, we will need another PR for that